reports/Arizona.md

```diff
@@ -1,3 +1,28 @@
+## Ajo
+
+### Indigenous protesters shot and tear gassed | October 12th
+
+Members of the O'odham Anti Border Collective staged a protest near Border Patrol checkpoint SR-85 in Arizona on Indigenous People's Day (October 12th). Protesters blocking the street were met with impact munitions (allegedly rubber bullets) and tear gas fired by officers from the Arizona Department of Public Safety.
+
+Demonstrations have centered on Quitobaquito Springs, a local freshwater source along the border of Organ Pipe National Monument, part of the traditional Hia-Ced O'odham homeland. Construction of the border wall puts access to this traditional resource at risk for indigenous people living in the area.
+
+Protesters say 8 adults were taken into custody, as well as several minors.
+
+tags: less-lethal, projectile, protester, rubber-bullet, shoot
+
+id: az-ajo-1
+
+geolocation: 32.2739671, -112.7419162
+
+**Links**
+
+* https://twitter.com/NoBayouBridge/status/1315773267014299653
+* https://twitter.com/maxie_adler/status/1315704218116861960
+* https://vimeo.com/467449845
```

Comment:
This video and the video on line 19 are the same, should one be struck?

Reply:
Nah it's just multiple links. The tweet is where we found it, the vimeo is the actual source